### PR TITLE
Update getting help guide

### PR DESCRIPTION
**Pull request by wsdjeg**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [ ] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [ ] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I have updated the [Following-HEAD](https://github.com/SpaceVim/SpaceVim/blob/master/wiki/en/Following-HEAD.md) page for this PR.
- [ ] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### Why this change is necessary and useful?

The getting help guide should be updated, I think it is useful source for new users. and it should help them find the answer more easily.

### CODE REVIEW

File: wiki/en/Getting-help.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+If you need help, ask your question in the [Gitter Chat](https://gitter.im/SpaceVim/SpaceVim), a member of the community will help you out.
+
+here is a simple guide for how to getting help about SpaceVim. Before asking question, please read the FAQ, the answer to your question maybe
```
Comment (jfemiani) [inline wiki/en/Getting-help.md]: English:  
* Either say "for how to get help with SpaceVim" or "for getting help with SpaceVim". 
* Either "before asking a question" or "before asking questions"
* s/maybe/may be/

File: wiki/en/Getting-help.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+listed there. 
+
+
+### Transitioning from origin vim config
```
Comment (jfemiani) [inline wiki/en/Getting-help.md]: * "from the original vim config".
* _I_ would prefer to say "to init.toml"

File: wiki/en/Getting-help.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+If you are not new to Vim, and you have familial with your own vim config, We recommend to use `vimcompatible` mode, In this mode, no origin key
```
Comment (jfemiani) [inline wiki/en/Getting-help.md]: * s/have familial/are familiar/
* s/We/we
* I _think_ you mean "no original key bindings will be overriden"

File: README.md [hunk 4]
```diff
@@ -26,6 +26,8 @@ SpaceVim is not just a vimrc but an ultimate Vim configuration, It contains many
 The last release is v0.8.0, check out [following-HEAD](https://github.com/SpaceVim/SpaceVim/wiki/Following-HEAD) page for what happened since last release.
 
 See the [quick start guide](https://spacevim.org/quick-start-guide/), [documentation](https://spacevim.org/documentation/) or the [available layers](http://spacevim.org/layers/) for more information.
+If you have any question about SpaceVim, please follow the [Getting Help guide](https://github.com/SpaceVim/SpaceVim/wiki/Getting-help).
```
Comment (jfemiani) [inline README.md]: * s/question/questions

Comment (wsdjeg) [reply README.md]: @jfemiani Thank your, I have fixed all of there.

**Pull request merged**